### PR TITLE
Add dry-run option for `config-case` 

### PR DESCRIPTION
**Pull request by fevac**
## Description

Related to: https://github.com/Clinical-Genomics/cg/issues/1665

### Added

- Dry-run option for `cg workflow rnafusion config-case`

### Fixed

- `cg workflow rnafusion run` does not set the action to `running` when the dry-run flag is set

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b add_dry_run_for_config_case -a
    ```

### How to test

- [x] Dry-run `cg workflow rnafusion config-case -d solidfawn`. Make sure that case folder is not created and no samplesheet is written.
- [x] Dry-run `cg workflow rnafusion start -d solidfawn`. Make sure that case folder is not created and no samplesheet is written.
- [x] Create a config for solidfawn, then: Dry-run `cg workflow rnafusion run -d solidfawn`. Make sure the following log doesn't appear in the output `Action running set for case solidfawn`

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/meta/workflow/nextflow_common.py [hunk 1]
```diff
@@ -61,9 +61,10 @@ def verify_analysis_finished(cls, case_id: str, root_dir: str) -> None:
             )
 
     @classmethod
-    def make_case_folder(cls, case_id: str, root_dir: str) -> None:
+    def make_case_folder(cls, case_id: str, root_dir: str, dry_run: bool = False) -> None:
         """Make the case folder where analysis should be located."""
-        os.makedirs(cls.get_case_path(case_id, root_dir), exist_ok=True)
+        if not dry_run:
+            os.makedirs(cls.get_case_path(case_id, root_dir), exist_ok=True)
```
Comment (henrikstranneheim) [inline cg/meta/workflow/nextflow_common.py:67]: Use named args

File: tests/cli/workflow/rnafusion/test_cli_rnafusion_compound_commands.py [hunk 2]
```diff
@@ -183,7 +183,18 @@ def test_store_available(
...
+    # Ensure the config is mocked to run compound command
+    Path.mkdir(
+        Path(
+            rnafusion_context.meta_apis["analysis_api"].get_case_config_path(case_id_success)
+        ).parent,
+        exist_ok=True,
+    )
+    Path(rnafusion_context.meta_apis["analysis_api"].get_case_config_path(case_id_success)).touch(
+        exist_ok=True
+    )
```
Comment (henrikstranneheim) [inline tests/cli/workflow/rnafusion/test_cli_rnafusion_compound_commands.py:196]: Should this be  a fixture instead for reuse by other RNAFusion CLI cmds and make the test easier to read?

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by fevac** [comment]
Tests output:

- [x] `cg workflow rnafusion config-case -d solidfawn`

Ends with: `Dry run: Samplesheet will not be written`

- [x] `cg workflow rnafusion start -d solidfawn`

Ends with: 
```
Dry run: Samplesheet will not be written
Case solidfawn exists in status db
Could not run analysis: No config file found for case solidfawn
Aborted!
```

- [x] `cg workflow rnafusion run -d solidfawn` with existing config

Ends with: `Dry run: process call will not be executed!!` and action is not set.

**Comment by fevac** [comment]
Deployed in prod:

```
$ bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-prod.sh -e P_main -t cg -b master -a

/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))

Collecting git+https://github.com/Clinical-Genomics/cg@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/cg /home/eva.caceres/tmp/pip-req-build-tm3kz0cj
  Cloning https://github.com/Clinical-Genomics/cg (to revision master) to ./tmp/pip-req-build-tm3kz0cj
  Resolved https://github.com/Clinical-Genomics/cg to commit e6d5c5d7b0eded19f843518d8c92f6e7e61a0693
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Building wheels for collected packages: cg
  Building wheel for cg (pyproject.toml): started
  Building wheel for cg (pyproject.toml): finished with status 'done'
  Created wheel for cg: filename=cg-24.0.2-py3-none-any.whl size=654751 sha256=7bd0c35a305e22318910da25c21d6bfddaae451d8f05b673d00e007369600ada
  Stored in directory: /home/eva.caceres/tmp/pip-ephem-wheel-cache-r0q84cwa/wheels/4b/55/66/90c6ed137f9153944f27209bd5c3790044b15cf24d2c7f573b
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 24.0.1
    Uninstalling cg-24.0.1:
      Successfully uninstalled cg-24.0.1
Successfully installed cg-24.0.2
Collecting git+https://github.com/Clinical-Genomics/cg@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/cg /home/eva.caceres/tmp/pip-req-build-7woznfc8
  Cloning https://github.com/Clinical-Genomics/cg (to revision master) to ./tmp/pip-req-build-7woznfc8
  Resolved https://github.com/Clinical-Genomics/cg to commit e6d5c5d7b0eded19f843518d8c92f6e7e61a0693
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Requirement already satisfied: typing-extensions in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (4.4.0)
Requirement already satisfied: Click<7.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (6.7)
Requirement already satisfied: google-auth in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.13.0)
Requirement already satisfied: cgmodels>=0.10.5 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.10.5)
Requirement already satisfied: petname in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.6)
Requirement already satisfied: ansi in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.3.6)
Requirement already satisfied: lxml in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (4.9.1)
Requirement already satisfied: pyyaml in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (6.0)
Requirement already satisfied: alembic in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.8.1)
Requirement already satisfied: coloredlogs in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (15.0.1)
Requirement already satisfied: pydantic in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.10.2)
Requirement already satisfied: requests-cache in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.9.6)
Requirement already satisfied: Flask-Dance in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (5.1.0)
Requirement already satisfied: sendmail-container in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.2.0)
Requirement already satisfied: paramiko in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.11.0)
Requirement already satisfied: pandas in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.3.5)
Requirement already satisfied: blinker in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.5)
Requirement already satisfied: wtforms<3.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.3.3)
Requirement already satisfied: werkzeug<1.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.16.1)
Requirement already satisfied: packaging in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (21.3)
Requirement already satisfied: Flask-Alchy in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.5.0)
Requirement already satisfied: genologics in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.4.6)
Requirement already satisfied: Flask-Admin in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.6.0)
Requirement already satisfied: Flask>=1.1.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.1.4)
Requirement already satisfied: Flask-CORS in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (3.0.10)
Requirement already satisfied: Flask-SQLAlchemy==2.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.1)
Requirement already satisfied: setuptools>=39.2.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (65.5.0)
Requirement already satisfied: marshmallow>3 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (3.18.0)
Requirement already satisfied: SQLAlchemy<1.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.3.24)
Requirement already satisfied: python-dateutil in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.8.2)
Requirement already satisfied: gunicorn in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (20.1.0)
Requirement already satisfied: markupsafe<2.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.0.1)
Requirement already satisfied: Alchy in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.2.2)
Requirement already satisfied: email-validator in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.3.0)
Requirement already satisfied: housekeeper>=2.5 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (3.2.0)
Requirement already satisfied: requests[security] in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (2.28.1)
Requirement already satisfied: openpyxl in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (3.0.10)
Requirement already satisfied: tabulate in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (0.9.0)
Requirement already satisfied: pymysql in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cg==24.0.2) (1.0.2)
Requirement already satisfied: dataclasses in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (0.6)
Requirement already satisfied: wcwidth in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (0.2.5)
Requirement already satisfied: attrs in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (22.1.0)
Requirement already satisfied: atomicwrites in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (1.4.1)
Requirement already satisfied: pytest in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (7.1.3)
Requirement already satisfied: colorama in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (0.4.5)
Requirement already satisfied: more-itertools in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (8.14.0)
Requirement already satisfied: pluggy in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (1.0.0)
Requirement already satisfied: coverage in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (6.5.0)
Requirement already satisfied: pyparsing in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (3.0.9)
Requirement already satisfied: pytest-cov in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (4.0.0)
Requirement already satisfied: py in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (1.11.0)
Requirement already satisfied: importlib-metadata in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (5.0.0)
Requirement already satisfied: zipp in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cgmodels>=0.10.5->cg==24.0.2) (3.9.0)
Requirement already satisfied: Jinja2<3.0,>=2.10.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from Flask>=1.1.1->cg==24.0.2) (2.11.3)
Requirement already satisfied: itsdangerous<2.0,>=0.24 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from Flask>=1.1.1->cg==24.0.2) (1.1.0)
Requirement already satisfied: rich in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper>=2.5->cg==24.0.2) (12.6.0)
Requirement already satisfied: ruamel.yaml in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper>=2.5->cg==24.0.2) (0.17.21)
Requirement already satisfied: pydash>=3.4.7 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from Alchy->cg==24.0.2) (5.1.1)
Requirement already satisfied: Mako in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from alembic->cg==24.0.2) (1.2.3)
Requirement already satisfied: importlib-resources in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from alembic->cg==24.0.2) (5.10.0)
Requirement already satisfied: humanfriendly>=9.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from coloredlogs->cg==24.0.2) (10.0)
Requirement already satisfied: dnspython>=1.15.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from email-validator->cg==24.0.2) (2.2.1)
Requirement already satisfied: idna>=2.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from email-validator->cg==24.0.2) (3.4)
Requirement already satisfied: Six in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from Flask-CORS->cg==24.0.2) (1.16.0)
Requirement already satisfied: oauthlib in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from Flask-Dance->cg==24.0.2) (3.2.2)
Requirement already satisfied: requests-oauthlib>=1.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from Flask-Dance->cg==24.0.2) (1.3.1)
Requirement already satisfied: urlobject in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from Flask-Dance->cg==24.0.2) (2.4.3)
Requirement already satisfied: rsa<5,>=3.1.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from google-auth->cg==24.0.2) (4.9)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from google-auth->cg==24.0.2) (5.2.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from google-auth->cg==24.0.2) (0.2.8)
Requirement already satisfied: et-xmlfile in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from openpyxl->cg==24.0.2) (1.1.0)
Requirement already satisfied: pytz>=2017.3 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from pandas->cg==24.0.2) (2022.4)
Requirement already satisfied: numpy>=1.17.3 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from pandas->cg==24.0.2) (1.21.6)
Requirement already satisfied: bcrypt>=3.1.3 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from paramiko->cg==24.0.2) (4.0.1)
Requirement already satisfied: cryptography>=2.5 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from paramiko->cg==24.0.2) (38.0.1)
Requirement already satisfied: pynacl>=1.0.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from paramiko->cg==24.0.2) (1.5.0)
Requirement already satisfied: url-normalize>=1.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from requests-cache->cg==24.0.2) (1.4.3)
Requirement already satisfied: urllib3>=1.25.5 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from requests-cache->cg==24.0.2) (1.26.12)
Requirement already satisfied: cattrs>=1.8 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from requests-cache->cg==24.0.2) (22.2.0)
Requirement already satisfied: appdirs>=1.4.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from requests-cache->cg==24.0.2) (1.4.4)
Requirement already satisfied: certifi>=2017.4.17 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from requests[security]->cg==24.0.2) (2022.9.24)
Requirement already satisfied: charset-normalizer<3,>=2 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from requests[security]->cg==24.0.2) (2.1.1)
Requirement already satisfied: python-multipart in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from sendmail-container->cg==24.0.2) (0.0.5)
Requirement already satisfied: fastapi in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from sendmail-container->cg==24.0.2) (0.85.1)
Requirement already satisfied: uvicorn in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from sendmail-container->cg==24.0.2) (0.5.1)
Requirement already satisfied: exceptiongroup in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cattrs>=1.8->requests-cache->cg==24.0.2) (1.0.0rc9)
Requirement already satisfied: cffi>=1.12 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cryptography>=2.5->paramiko->cg==24.0.2) (1.15.1)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from pyasn1-modules>=0.2.1->google-auth->cg==24.0.2) (0.4.8)
Requirement already satisfied: starlette==0.20.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from fastapi->sendmail-container->cg==24.0.2) (0.20.4)
Requirement already satisfied: anyio<5,>=3.4.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from starlette==0.20.4->fastapi->sendmail-container->cg==24.0.2) (3.6.1)
Requirement already satisfied: python-dotenv>=0.10.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from pydantic->cg==24.0.2) (0.21.0)
Requirement already satisfied: iniconfig in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from pytest->cgmodels>=0.10.5->cg==24.0.2) (1.1.1)
Requirement already satisfied: tomli>=1.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from pytest->cgmodels>=0.10.5->cg==24.0.2) (2.0.1)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper>=2.5->cg==24.0.2) (2.13.0)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper>=2.5->cg==24.0.2) (0.9.1)
Requirement already satisfied: ruamel.yaml.clib>=0.2.6 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from ruamel.yaml->housekeeper>=2.5->cg==24.0.2) (0.2.6)
Requirement already satisfied: h11 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from uvicorn->sendmail-container->cg==24.0.2) (0.14.0)
Requirement already satisfied: uvloop in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from uvicorn->sendmail-container->cg==24.0.2) (0.17.0)
Requirement already satisfied: httptools in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from uvicorn->sendmail-container->cg==24.0.2) (0.5.0)
Requirement already satisfied: websockets>=6.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from uvicorn->sendmail-container->cg==24.0.2) (10.3)
Requirement already satisfied: pycparser in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from cffi>=1.12->cryptography>=2.5->paramiko->cg==24.0.2) (2.21)
Requirement already satisfied: sniffio>=1.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from anyio<5,>=3.4.0->starlette==0.20.4->fastapi->sendmail-container->cg==24.0.2) (1.3.0)
/home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))

remote: Enumerating objects: 109, done.
remote: Counting objects: 100% (109/109), done.
remote: Compressing objects: 100% (46/46), done.
remote: Total 109 (delta 69), reused 80 (delta 61), pack-reused 0
Receiving objects: 100% (109/109), 29.87 KiB | 0 bytes/s, done.
Resolving deltas: 100% (69/69), completed with 44 local objects.
From https://github.com/Clinical-Genomics/cg
   909878d..e6d5c5d  master     -> origin/master
 * [new branch]      1698-feat-cancel-samples -> origin/1698-feat-cancel-samples
   4f68abc..e92c604  add-microsalt-qc -> origin/add-microsalt-qc
   d7064d8..97ada49  include_bundle_and_update_spring_metadata -> origin/include_bundle_and_update_spring_metadata
   f03f61e..960d799  verify-integrity-ext-resource -> origin/verify-integrity-ext-resource
 * [new tag]         v24.0.2    -> v24.0.2
Already on 'master'
Your branch is behind 'origin/master' by 2 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)
Updating 909878d..e6d5c5d
Fast-forward
 .bumpversion.cfg                                                     |  2 +-
 cg/__init__.py                                                       |  2 +-
 cg/cli/workflow/rnafusion/base.py                                    | 71 +++++++++++++++++++++++++++++++++++------------------------------------
 cg/constants/constants.py                                            |  2 ++
 cg/meta/workflow/analysis.py                                         | 13 ++++++-------
 cg/meta/workflow/nextflow_common.py                                  |  5 +++--
 cg/meta/workflow/rnafusion.py                                        | 17 ++++++++++++++---
 setup.py                                                             |  2 +-
 tests/cli/workflow/rnafusion/test_cli_rnafusion_compound_commands.py | 12 +++++++-----
 tests/cli/workflow/rnafusion/test_cli_rnafusion_run.py               | 12 ++++--------
 10 files changed, 74 insertions(+), 64 deletions(-)
/home/eva.caceres
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/eva.caceres
done.
Log deploy... done.
cg, version 24.0.2
```

**Pull request merged**